README.md

```diff
@@ -1,7 +1,15 @@
 # Olin.js 2015
 
-To generate `index.html`:
+## Dependencies
+Ensure that jade is installed (`npm install -g jade`).
 
+If the file `token.txt` does not exist in the root of the directory, create it, then follow [these instructions](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) to create an access token. Paste the token into the file `token.txt`.
+
+## Updating content
+To update the content of the site, edit `jade/in.json`.
```

Comment:
Feel free to change the names/locations of files if you think it would improve clarity (I didn't think this part was particularly clear when I made it).